x-pack/plugins/watcher/public/application/components/watch_state_badge.tsx

```diff
@@ -0,0 +1,34 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import React from 'react';
+import { EuiIcon, EuiFlexGroup, EuiFlexItem, EuiText } from '@elastic/eui';
+import { WATCH_STATES } from '../../../common/constants';
+
+interface Props {
+  state: string;
+  size?: 'xs' | 's' | 'm';
+}
+
+const stateToIconMap = {
+  [WATCH_STATES.ACTIVE]: <EuiIcon type="check" color="success" />,
```

Comment:
What do you think about changing the `Active` icon to a green dot? Maybe not a big deal, but we're also using the `check` icon for the `Condition met` column on the history page, so sometimes it's displayed for both columns.

<img width="1103" alt="Screen Shot 2022-08-15 at 3 17 30 PM" src="https://user-images.githubusercontent.com/5226211/184702031-ca29232f-5719-4869-98ff-734b54611a7e.png">

Reply:
Great idea, will do.